### ISSUE TITLE
Aztec: Review Blog Picker's Behavior in Single Site Mode

### ISSUE THREAD

**Issue by jleandroperez** [wordpress-mobile/WordPress-iOS#6503]
```
Should it ever be possible to select the block picker in single site mode? This looks way wrong as it is.

Also, why cancel editing? That feels like a line that should be removed indeed.

I'd consider replacing the cancelEditing() line with an assertion failure (unless I'm missing something else).
```

cc @diegoreymendez

**Comment by diegoreymendez** [comment]
Is this linked to any specific PR?  I was just wondering what the resolution was.

**Comment by jleandroperez** [comment]
@diegoreymendez Yes sir! Addressed in PR #6541. Based on your feedback, from now on:

- The blog picker won't be touchable, when in single site mode.
- I've added an assertion in the Picker's delegate, to ensure that code is never executed when in single site mode.

If you'd like to fine tune further, please, just let me know. Will be more than happy to hack it out!

**Comment by diegoreymendez** [comment]
Thanks @jleandroperez!   As usual: awesome work!
